contao/templates/rsce/rsce_hyperlink_list.html5

```diff
@@ -2,11 +2,17 @@
 
 <?php $this->block('content'); ?>
 
+<?php $previewMode = Contao\System::getContainer()->get('contao.security.token_checker')->isPreviewMode() ?>
+
 <ul class="c_list<?=$this->styleManager->prepare('icon', ['icon'])->format(' li-icon')?><?=$this->styleManager->prepare('list')->format(' %s')?>">
     <?php foreach ($this->list as $item): ?>
+    <?php if(!$item->invisible || $previewMode): ?>
+    <div class="ce_rsce_hyperlink<?= !$item->cssClass ?: ' '.$item->cssClass ?><?= $this->styleManager->prepare('layout', ['boxedListElement'])->format(' %s')?> block">
```

Comment:
If no class has been defined, this notation will cause a `1` (boolean value) to be inserted as the class.

I think it should look like this:
`$cssClass ? ' ' . $cssClass : ''`